scene/gui/popup_menu.cpp

```diff
@@ -612,6 +612,9 @@ void PopupMenu::_draw_items() {
 
 		Color icon_color(1, 1, 1, items[i].disabled && !items[i].separator ? 0.5 : 1);
 
+		// Multiply icon color by the custom modulate color
```

Comment:
```suggestion
		// Multiply icon color by the custom modulate color.
```
Although I think this comment isn't necessary.

Reply:
Thanks, I agree the comment is unnecessary - will go ahead and remove it.